### PR TITLE
PD-46 - Refactor users page

### PR DESCRIPTION
**Pull request by WashingtonKK**
# What type of PR is this?

### CODE REVIEW

File: docs/users.md
Comment (SammyOina) [inline docs/users.md:1]: use images where possible, for example user creation/regsitration

Comment (SammyOina) [reply docs/users.md:1]: document login and other user operations

Comment (SammyOina) [reply docs/users.md:1]: document changing user passwords, account deactivation and removing themselves from workspaces etc

**Review by SammyOina — CHANGES_REQUESTED** [review]

File: docs/users.md
Comment (SammyOina) [inline docs/users.md:1]: Update the relevant section that is plainly curl commands to incoorporate UI images

**Review by SammyOina — CHANGES_REQUESTED** [review]

File: docs/users.md
Comment (WashingtonKK) [reply docs/users.md:1]: These are all documented on the section below. The user can not obtain token from ui so this section will have to remain without images

Comment (SammyOina) [inline docs/users.md:1]: add login to create token section

**Review by SammyOina — CHANGES_REQUESTED** [review]

**Review by SammyOina — APPROVED** [review]

**Pull request merged**